src/cache/gcs.rs

```diff
@@ -143,16 +143,8 @@ pub struct GCSCredentialProvider {
 
 #[derive(Debug, Deserialize)]
 struct ServiceAccountKey {
-    #[serde(rename = "type")]
```

Comment:
It's not clear to me from the serde-json docs--will serde deserialize a struct OK if there are additional fields in the JSON? If so it might be worth a comment here mentioning that there are additional fields that we don't need.